ui/packages/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.less

```diff
@@ -1,38 +1,33 @@
 @{customElementNamespace}query-interactions {
   .query-interaction,
-  .interaction-icon,
   .interaction-text {
     line-height: @minimumButtonSize;
     height: @minimumButtonSize;
   }
 
   .query-interaction {
     white-space: nowrap;
-    padding: 0px 10px;
+    padding: 0px @largeSpacing;
     cursor: pointer;
-    overflow: hidden;
-  }
-
-  .interaction-icon {
-    text-align: center;
-    width: @minimumButtonSize;
   }
 
-  .interaction-icon,
   .interaction-text {
     display: inline-block;
     vertical-align: top;
-  }
-
-  .interaction-text {
-    width: ~'calc(100% - @{minimumButtonSize})';
     overflow: hidden;
     text-overflow: ellipsis;
+    overflow: hidden;
+    width: 210px;
```

Comment:
Will this be compatible with small screen sizes?

Reply:
As DDF currently exists it is not responsive. There may be a more elegant way to accomplish this width though. I used this width to match the width of other menus that are related to it.

Reply:
Remove this width